staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors_test.go

```diff
@@ -169,7 +170,140 @@ func TestErrListFilter(t *testing.T) {
 func TestNotSupported(t *testing.T) {
 	notSupported := NotSupported(NewPath("f"), "v", []string{"a", "b", "c"})
 	expected := `Unsupported value: "v": supported values: "a", "b", "c"`
+
 	if notSupported.ErrorBody() != expected {
 		t.Errorf("Expected: %s\n, but got: %s\n", expected, notSupported.ErrorBody())
 	}
 }
+
+func TestIs(t *testing.T) {
+
+	testCases := []struct {
+		name     string
+		gotErr   *Error
```

Comment:
This is not testing much, and variable names are confusing. I
would suggest:

```golang
struct {
  name string
  gotErr error
  cmpErr Err
  wantIs bool
}
```

And `gotErr` should include aggregate errors and wrapping errors.

Reply:
rename `gotErr` to `targetErr`